routes/web.php

```diff
@@ -16,3 +19,15 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::prefix('user')->controller(ControllersUserController::class)->group(function () {
+    Route::get('/', 'index')->name('index');
```

Comment:
```suggestion
    Route::get('/', 'index')->name('user.index');
```
hoặc viết prefix route name trong group nhé